browser/src/slideshow/engine/AnimatedElement.ts

```diff
@@ -619,11 +596,9 @@ class AnimatedElement {
 		]);
 		renderingContext.setTransform(transform);
 
-		const aElement = applied
-			? this.tfCanvas.transferToImageBitmap()
-			: this.aBaseElement;
+		const aElement = this.aBaseElement;
 		console.debug(
-			`AnimatedElement(${this.sId}).updateLayer:
+			`AnimatedElement(${this.sId}).renerLayer2d:
```

Comment:
I guess this should've been renderLayer2d

Reply:
yep, thanks :-)